src/main/resources/type-mapping.yml

```diff
@@ -81,13 +81,37 @@ date:
   impala: bigint
   parquet: timestamp
   avro: bigint
-  snowflake: timestamp
```

Comment:
I think it can be future work, but this is where it'd be nice to be able to override these values. For SQLMorph, we give the user the ability to map oracle date to date, timestamp, or datetime because oracle date can store all of these.

Perhaps we can today by changing the template afterwards?

Reply:
I guess users could also update the type mapping themselves too.